src/state/user/reducer.ts

```diff
@@ -140,14 +140,6 @@ const userSlice = createSlice({
       state.tokens[serializedToken.chainId][serializedToken.address] = serializedToken
       state.timestamp = currentTimestamp()
     },
-    removeSerializedToken(state, { payload: { address, chainId } }) {
```

Comment:
Looks like we have "addSerializedToken" too. Looks like we needed to remove them at some point. Since I am not sure what's a serialised token in this case, hard for me to judge whether this is/may be needed in the future.

Reply:
unclear to me too. but i will make a task to come back to look at this... we probably don't need addSerializedToken